cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -0,0 +1,529 @@
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_MESSAGE_CONNVERTER = "KAFKA_BRIDGE_AMQP_MESSAGE_CONNVERTER";
+
+    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_ENABLED = "KAFKA_BRIDGE_HTTP_ENABLED";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_HOST = "KAFKA_BRIDGE_HTTP_HOST";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_PORT = "KAFKA_BRIDGE_HTTP_PORT";
+
+    private KafkaBridgeTls tls;
+    private CertAndKeySecretSource tlsAuthCertAndKey;
+    private PasswordSecretSource passwordSecret;
+    private String username;
+    private String saslMechanism;
+    private KafkaBridgeHttpConfig http;
+    private boolean httpEnabled = false;
+    private boolean amqpEnabled = false;
+    private String bootstrapServers;
+    private KafkaBridgeConsumerSpec kafkaBridgeConsumer;
+    private KafkaBridgeProducerSpec kafkaBridgeProducer;
+
+    /**
+     * Constructor
+     *
+     * @param namespace Kubernetes/OpenShift namespace where Kafka Bridge cluster resources are going to be created
+     * @param cluster   overall cluster name
+     * @param labels    labels to add to the cluster
+     */
+    protected KafkaBridgeCluster(String namespace, String cluster, Labels labels) {
+        super(namespace, cluster, labels);
+        this.name = kafkaBridgeClusterName(cluster);
+        this.serviceName = name + "-service";
+        this.ancillaryConfigName = logAndMetricsConfigName(cluster);
+        this.replicas = DEFAULT_REPLICAS;
+        this.readinessPath = "/ready";
+        this.livenessPath = "/healthy";
+        this.livenessProbeOptions = DEFAULT_HEALTHCHECK_OPTIONS;
+        this.readinessProbeOptions = DEFAULT_HEALTHCHECK_OPTIONS;
+        this.isMetricsEnabled = DEFAULT_KAFKA_BRIDGE_METRICS_ENABLED;
+
+        this.mountPath = "/var/lib/bridge";
+        this.logAndMetricsConfigVolumeName = "kafka-metrics-and-logging";
+        this.logAndMetricsConfigMountPath = "/opt/bridge/custom-config/";
+    }
+
+    public static String kafkaBridgeClusterName(String cluster) {
+        return cluster + KafkaBridgeCluster.NAME_SUFFIX;
+    }
+
+    public static String serviceName(String cluster) {
+        return cluster + KafkaBridgeCluster.SERVICE_NAME_SUFFIX;
+    }
+
+    public static String logAndMetricsConfigName(String cluster) {
+        return cluster + KafkaBridgeCluster.METRICS_AND_LOG_CONFIG_SUFFIX;
+    }
+
+    public static KafkaBridgeCluster fromCrd(KafkaBridge kafkaBridge, KafkaVersion.Lookup versions) {
+
+        KafkaBridgeCluster kafkaBridgeCluster = new KafkaBridgeCluster(kafkaBridge.getMetadata().getNamespace(),
+                kafkaBridge.getMetadata().getName(), Labels.fromResource(kafkaBridge).withKind(kafkaBridge.getKind()));
+
+        KafkaBridgeSpec spec = kafkaBridge.getSpec();
+        kafkaBridgeCluster.setResources(spec.getResources());
+        kafkaBridgeCluster.setLogging(spec.getLogging());
+        kafkaBridgeCluster.setJvmOptions(spec.getJvmOptions());
```

Comment:
I think the JVM options are currently designed mainly for Kafka and set the Kafka specific JVM options. But we can use the gcLogging from there:

```suggestion
        kafkaBridgeCluster.setGcLoggingEnabled(spec.getJvmOptions() == null ? true : spec.getJvmOptions().isGcLoggingEnabled());
```